websites/W/Weverse/presence.ts

```diff
@@ -0,0 +1,181 @@
+class Weverse extends Presence {
+	constructor(options: PresenceOptions) {
+		super(options);
+	}
+
+	getArtistName(): string {
+		return (
+			document
+				.querySelector(".LiveArtistProfileView_name_item__8W66y")
+				?.textContent?.trim() ?? "Unknown Artist"
+		);
+	}
+
+	getStreamTitle(): string {
+		const titleElement = document.querySelector("h2.TitleView_title__SSnHb");
```

Comment:
For better future support, consider using a selector without computer generated classes. For example,
```ts
"[class*=LiveArtistProfileView_name_item]"
```

Reply:
check my new commits